### DISCUSSION TITLE
希望 ConfigProvider 可以支持传入 theme 参数进行动态主题设置

### DISCUSSION THREAD

**Discussion by wkl007** [element-plus/element-plus#7644] [Ideas]
目前 Element Plus 要实现动态主题换肤方式不太友好，希望 ConfigProvider 可以支持传入 theme 参数进行动态主题设置，类似于 Ant Design 的实现方式：https://ant-design.gitee.io/docs/react/customize-theme-variable-cn

**Comment by sxzz** [comment]
只需要添加一个 dark CSS 类名即可，与 config-provider 差不多吧
